src/components/common/dashboard/Navbar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Link, Button, Flex, Heading, Icon, Image, VStack } from '@chakra-ui/react';
+import { FiUsers } from 'react-icons/fi';
+import { IoHomeOutline, IoPodiumOutline } from 'react-icons/io5';
+import { Link as RouterLink, NavLink } from 'react-router-dom';
+import Logo from '../../../assets/logoWhite.png';
+
+const NavItem = props => (
+  <Link as={NavLink} to={props.to} _activeLink={{ color: 'white' }}>
+    <Heading size="h4" as="span">
+      <Flex>
+        <Icon as={props.icon} mr={2} />
+        {props.children}
+      </Flex>
+    </Heading>
+  </Link>
+);
+
+const Navbar = props => (
+  <Flex
+    as="nav"
+    bg="background.secondary"
+    direction="column"
+    h="100%"
+    px={12}
+    py={20}
+    justify="space-between"
+    overflowX="hidden"
+    textTransform="capitalize"
+    role="navigation"
+  >
+    <VStack color="gray.400" spacing={10} align="flex-start">
+      <Image src={Logo} alt="Raising The Roof Logo" w="100%" />
+      <NavItem to="/home" icon={IoHomeOutline}>
+        home
+      </NavItem>
+      <NavItem to="/leaderboard" icon={IoPodiumOutline}>
+        leaderboard
+      </NavItem>
+      <NavItem to="/team" icon={FiUsers}>
+        team
+      </NavItem>
+      <Button color="black" w="100%">
+        Share Leaderboard
+      </Button>
+    </VStack>
+    <Link as={RouterLink} to="/logout">
+      <Heading as="span" color="white" size="h4">
```

Comment:
Kind of nitpicky but, maybe we want to add some margin/padding between the share leaderboard button and the logout button When I make my window really small it looks like this: 
![image](https://user-images.githubusercontent.com/36828558/109433138-d2b86d00-79dc-11eb-99e9-fa869cac474f.png)

Reply:
Also kind of a nitpick: should it be "Log Out" or "Logout"